cmd/server/grpc/main.go

```diff
@@ -351,3 +365,56 @@ func (s *server) Signal(stream pb.SFU_SignalServer) error {
 		}
 	}
 }
+
+// Relay endpoint implementation
+func (s *Server) Relay(ctx context.Context, request *pb.RelayRequest) (*pb.RelayResponse, error) {
+
+	conn, err := grpc.Dial(request.Sfu, grpc.WithInsecure(), grpc.WithBlock())
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "%s", err)
+	}
+	defer conn.Close()
+
+	sfuClient := pb.NewSFUClient(conn)
+
+	client, err := sfuClient.Signal(ctx)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "%s", err)
+	}
+
+	me := sfu.MediaEngine{}
+	me.RegisterDefaultCodecs()
+
+	t, err := sfu.NewWebRTCTransport(ctx, &sfu.Session{}, me, sfu.WebRTCTransportConfig{})
```

Comment:
We should use the session if it exists on the sfu, otherwise create a new session with `sid`